README.md

```diff
@@ -1,2 +1,68 @@
 # app.js
-GitHub Apps toolset for Node.js
+
+> Official GitHub Platform for GitHub Apps in Node.js
+
+## GitHub Apps
+
+`@octokit/app.js` allows to authenticate as GitHub Apps, installations, listen
+to webhooks, and respond to them using the GitHub API.
+
+### Authenticating as an App
+
+In order to authenticate as a GitHub App, you need to generate a Private Key
+and use it to sign a JSON Web Token (jwt) and encode it. 
+
+```js
+const App = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const appAuth = await app.appAuth()
```

Comment:
maybe add to the example how `appAuth` would now be used to send an authenticated request? You could use a 3rd party request library like [got](https://www.npmjs.com/package/got) or the current `@octokit/rest`. 

The example request could be to list installations of the app (https://developer.github.com/v3/apps/#find-installations)